### PR TITLE
Test scenario for searching oppo and verifying products

### PR DESCRIPTION
**Pull request by ShruthiNiranjan**

### CODE REVIEW

File: src/test/java/SalesforceModules/QuoteUtil.java [hunk 1]
```diff
@@ -1253,4 +1253,138 @@ public static void deleteQuote_billing(String logStep) throws Exception {
         KeywordUtil.click(QuoteObject.deleteButtonClicked, logStep);
     }
 
-}
+    /**
+     * Create new quote combining all methods in billing section
```
Comment (snaphray) [inline src/test/java/SalesforceModules/QuoteUtil.java]: is it creating or editing the quote billing as from method name it looks like it is editing the quote

File: src/test/java/step_definitions/SalesForceBilling.java [hunk 2]
```diff
@@ -83,5 +85,39 @@ public void create_new_quote_billing() throws Exception{
...
+        dataMap = ExcelDataUtil.getTestDataWithTestCaseID("Salesforce", "TestData1");
+        QuoteUtil.goToQuoteAndChangeStatus_InReview_billing((dataMap.get("InReviewStatus")), "Quote status is changed to In Review");
+        QuoteUtil.goToQuoteAndChangeStatus_Approved_billing((dataMap.get("ApprovedStatus")), "Quote status is changed to Approved");
+        QuoteUtil.goToQuoteAndCreateOrder_billing("Order generated and activated");
+        QuoteUtil.activateOrder_billing();
+        QuoteUtil.changeStatusAsDraft_billing((dataMap.get("changeStatusAsDraft")), "change order status as Draft ");
+            }
+
+    @When("^Create new contract in billing section$")
+    public void create_new_contract() throws Exception{
```
Comment (snaphray) [inline src/test/java/step_definitions/SalesForceBilling.java]: please follow same pattern for naming the method, either include the '_' or don't include it

**Review by snaphray — CHANGES_REQUESTED** [review]
comments proivided

File: src/test/java/step_definitions/SalesForceBilling.java (hunk 2 above)
Comment (ShruthiNiranjan) [reply src/test/java/step_definitions/SalesForceBilling.java]: done

File: src/test/java/SalesforceModules/QuoteUtil.java (hunk 1 above)
Comment (ShruthiNiranjan) [reply src/test/java/SalesforceModules/QuoteUtil.java]: done

**Review by snaphray — APPROVED** [review]
Code looks fine now

**Pull request merged**